app/main.py

```diff
@@ -1 +1,16 @@
-# write your code here
+def copy_file(command: str) -> None:
+    command_name, file_name, new_name = command.split(" ")
```

Comment:
check if command has 3 arguments before it

Reply:
Not fixed

Reply:
Also " " is default due to split